smact/tests/test_structure.py

```
@@ -642,24 +644,37 @@ def test_parse_mprest_exception_path():
+
+
 @pytest.fixture(scope="module")
 def cation_mutator_data():
     test_struct = SmactStructure.from_file(TEST_POSCAR)
 
     test_mutator = CationMutator.from_json(lambda_json=TEST_LAMBDA_JSON)
-    test_pymatgen_mutator = CationMutator.from_json(lambda_json=None, alpha=lambda _x, _y: -5)
+    default_mutator = CationMutator.from_json(lambda_json=None, alpha=lambda _x, _y: -5)
 
     # 5 random test species -> 5! test pairs (seeded for determinism)
     rng = random.Random(42)
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)